pytest.ini

```
@@ -0,0 +1,3 @@
+[pytest]
+filterwarnings =
+    ignore::DeprecationWarning
```

Do we really want to do this? Thinking of e.g. pymongo where the API seems to change between minor versions quite frequently... Instead we could filter out specific warnings (I see the py38 travis run produces lots of errors), e.g.

```
[pytest]
filterwarnings = 
    ignore:*PY_SSIZE_T_CLEAN will be required for '#' formats*:DeprecationWarning
    ignore:*"@coroutine" decorator is deprecated since Python 3.8, use "async def" instead*:DeprecationWarning
```

(not sure how to do that suggested change thing I've seen you do?)

---

(See the edit of this message):
```suggestion
    This is a suggested change
```

---

Concerning the actual comment:
I will implement the change and give you authorship of the commit.
I too would not like to hit too broadly on this, so being more precise would be good. It just doesn't make sense to have warnings about `django` and their "wrong" use of importing `collections.abc` etc..

---

That's really useful, thanks.